src/command.rs

```diff
@@ -560,6 +568,33 @@ impl App {
 
                 crate::bounded::set::check(&mut m, &universe, depth, compress_traces.into());
             }
+            Command::SatCheck { depth, .. } => {
+                let mut universe = HashMap::new();
+                let stdin = std::io::stdin();
+                for sort in &m.signature.sorts {
+                    println!("how large should {} be?", sort);
+                    let input = stdin
+                        .lock()
+                        .lines()
+                        .next()
+                        .expect("no next line")
+                        .expect("could not read next line")
+                        .parse::<usize>();
+                    match input {
+                        Ok(input) => {
+                            universe.insert(sort.clone(), input);
+                        }
+                        Err(_) => {
+                            eprintln!("could not parse input as a number");
+                            process::exit(1);
+                        }
+                    }
+                }
+                match crate::bounded::sat::check(&mut m, &universe, depth) {
+                    Ok(result) => println!("answer: {:?}", result),
```

Comment:
This prints "answer: Unknown" which is technically accurate but I'd prefer a more helpful result, like "safe up to _" (which is what I think that means).